tmt/steps/execute/__init__.py

```diff
@@ -206,6 +209,14 @@ def check_files_path(self) -> Path:
 
         return self.path / CHECK_DATA
 
+    @functools.cached_property
+    def submission_log_path(self) -> Path:
+        """
+        A path to log containing submitted files paths
+        """
+
+        return self.test_data_path / SUBMITTED_FILES_FILENAME
```

Comment:
The test data path should be reserved solely for stuff produced by the test itself (to prevent any file name collisions). Can we use the test invocation path instead?